lib/node_modules/@stdlib/blas/ext/base/dnannsumpw/src/main.c

```diff
@@ -17,25 +17,30 @@
 */
 
 #include "stdlib/blas/ext/base/dnannsumpw.h"
-#include <stdint.h>
+#include "stdlib/strided/base/stride2offset.h"
+#include "stdlib/math/base/assert/is_nan.h"
+#include "stdlib/blas/base/shared.h"
 
 /**
 * Computes the sum of double-precision floating-point strided array elements, ignoring `NaN` values and using pairwise summation.
 *
-* @param N       number of indexed elements
-* @param X       input array
-* @param stride  stride length
-* @param n       pointer for storing the number of non-NaN elements
-* @return        output value
+* @param N        number of indexed elements
+* @param X        input array
+* @param strideX  stride length
+* @param offsetX  starting index
+* @param n        pointer for storing the number of non-NaN elements
+* @return         output value
 */
-static double sumpw( const int64_t N, const double *X, const int64_t stride, int64_t *n ) {
-	double *xp1;
-	double *xp2;
+static double API_SUFFIX(stdlib_strided_sumpw)( const CBLAS_INT N, const double *X, const CBLAS_INT strideX, const CBLAS_INT offsetX, CBLAS_INT *n ) {
```

Comment:
@headlessNode Why was this function name changed? `sumpw` was a static function (i.e., internal to the translation unit). And because it is private, it doesn't need an API_SUFFIX. The `CBLAS_INT` changes are, however, necessary.